packages/perseus/src/widgets/expression.tsx

```diff
@@ -24,6 +25,17 @@ import type {
 } from "../perseus-types";
 import type {PerseusScore, WidgetExports, WidgetProps} from "../types";
 
+const sendExpressionEvaluatedEvent = (
+    result: "correct" | "incorrect" | "invalid",
+) => {
+    getDependencies().analytics.sendEvent({
+        eventType: "ContentLibraryMathInputBoxEvaluated",
+        eventSchemaVersion: 2,
+        virtualKeypadVersion: "PERSEUS_MATH_INPUT", // STOPSHIP: Or do we push that onto host application?
+        evaluationResult: result,
+    });
```

Comment:
Does this need to be on `PerseusDependencies`?  If we only ever need to call `sendEvent` from React components we could use `APIOptions` or even a Context.

Reply:
I updated the PR description, but I'll re-comment here specifically.  I'm opting to add this to `PerseusDependencies` because adding a new React Context dependency model means we'd now have two dependency models. I'm hesitant to take on the work to completely migrate everything (especially given that I don't think any of the original reasons we introduced `PerseusDependencies` have been resolved).  I also don't really want to have the migration started but not completed for the indefinite future (we don't really have bandwidth to dedicate to an infrastructure change like this right now).